packages/contracts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cabindao/nft-passport-contracts",
-  "version": "0.9.1",
+  "version": "0.10.0",
```

Comment:
forgot to version bump during the Rinkeby/Goerli PR - this will publish those contract addresses to npm